skyportal/models.py

```diff
@@ -2517,6 +2517,9 @@ class Spectrum(Base):
         doc="Median UTC ISO time stamp of the exposure or exposures in which the Spectrum was acquired.",
     )
     origin = sa.Column(sa.String, nullable=True, doc="Origin of the spectrum.")
+    type = sa.Column(
+        sa.String, nullable=True, doc="Type of spectrum: source, host, etc. "
```

Comment:
If we're going to enforce a default value at the handler level (that doesn't appear to be committed here yet), we should make this column non-nullable. We can also provide a default value here in the SQLA definition.

Reply:
Agreed.